src/SearchResults/index.js

```diff
@@ -224,6 +224,8 @@ function findMatchingHierarchicalFacetFromAttributeName(hierarchicalFacets, hier
 function SearchResults(state, results) {
   var mainSubResponse = results[0];
 
+  this._raw = mainSubResponse;
```

Comment:
Since it's gonna be available, what about we add all the results instead of just the first one?
```js
this._rawResults = results;
```